lib/states/az/mesa.R

```diff
@@ -49,6 +97,8 @@ clean <- function(d, helpers) {
       # TODO(phoebe): can we get clearer definitions of ped vs veh?
       # https://app.asana.com/0/456927885748233/571408593843006
       type = if_else(
+        # all of the parking charges with this prefix appear to be parking-related, 
+        # are these not considered vehicle? i guess we consider them NA?  
```

Comment:
I'm not sure. @danjenson Do you have any insight? (Feel free to say, "it's been too long and I forgot.")

Reply:
We can talk about this a little at our meeting. If we can't get it resolved, I think it's fine to stick with whatever Dan and Amy did.

Reply:
I think we should go with the original classification system that Dan and Amy vetted.